src/pages/Events/event-details.html

```diff
@@ -0,0 +1,477 @@
+              <i class="menu-icon mdi mdi-file-document-multiple-outline"></i>
+              <span class="menu-title">Articles</span>
+              <i class="menu-arrow"></i>
+            </a>
+          </li>
+          <!------------------------------------------------Event Category--------------------------------------------------->
+          <li class="nav-item nav-category">Event</li>
+          <li class="nav-item">
+            <a class="nav-link" data-bs-toggle="collapse" href="#auth" aria-expanded="false" aria-controls="auth">
+              <i class="menu-icon mdi mdi-calendar-plus"></i>
+              <span class="menu-title">Add Event</span>
+              <i class="menu-arrow"></i>
+            </a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link"  href="../Events/events.html" aria-expanded="false" aria-controls="auth">
+              <i class="menu-icon mdi mdi-calendar-multiple"></i>
+              <span class="menu-title">Events</span>
+              <i class="menu-arrow"></i>
+            </a>
+          </li>
+          <!---------------------------------------------------Jobs Category------------------------------------------------->
+          <li class="nav-item nav-category">Jobs</li>
+          <li class="nav-item">
+            <a class="nav-link" data-bs-toggle="collapse" href="#auth" aria-expanded="false" aria-controls="auth">
+              <i class="menu-icon mdi  mdi-earth-box-plus"></i>
+              <span class="menu-title">Add Categories</span>
+              <i class="menu-arrow"></i>
+            </a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" data-bs-toggle="collapse" href="#auth" aria-expanded="false" aria-controls="auth">
+              <i class="menu-icon mdi mdi-earth-box"></i>
+              <span class="menu-title">Categories</span>
+              <i class="menu-arrow"></i>
+            </a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link"  href="../../pages/jobs/jobs.html" aria-expanded="false" aria-controls="jobs">
+              <i class="menu-icon mdi  mdi-briefcase"></i>
+              <span class="menu-title">Jobs</span>
+              <i class="menu-arrow"></i>
+            </a>
+          </li>
+          <!------------------------------------------------------------------------------------------->
+          <li class="nav-item nav-category">help</li>
+          <li class="nav-item">
+            <a class="nav-link" href="http://bootstrapdash.com/demo/star-admin2-free/docs/documentation.html">
+              <i class="menu-icon mdi mdi-file-document"></i>
+              <span class="menu-title">Documentation</span>
+            </a>
+          </li>
+        </ul>
+      </nav>
+      <!-- partial -->
+      <div class="main-panel">
+        <div class="content-wrapper">
+            <h1 class="pt-5">Event Details</h1>
```

Comment:
This is the first thing, i have noticed in your code.

Take it as an advice from me, don't ever make two h1 tag in your design, This is considered as bad practice. 
A little trick, if you want to display it as h1 use bootstrap class. 

Actually i like to display it a little smaller, The following code snippet looks much better
 ```html
<h2 class="pt-5  fw-bold">Event Details</h2>
```